cmd/bnscli/cmd_escrow.go

```diff
@@ -23,7 +23,7 @@ Create a transaction for releasing funds from given escrow.
 		escrowFl = flSeq(fl, "escrow", "", "An ID of an escrow that is to be released.")
 		amountFl = flCoin(fl, "amount", "", "Optional amount that is to be transferred from the escrow. The whole escrow hold amount is used if no value is provided.")
 	)
-	fl.Parse(args)
+	_ = fl.Parse(args)
```

Comment:
Same here... Return parse error

Reply:
there is no reason, as we specify panicOnParse, but we can return it as well.